packet_forwarder/src/lora_pkt_fwd.c

```diff
@@ -1140,6 +1140,10 @@ static int send_tx_ack(uint8_t token_h, uint8_t token_l, enum jit_error_e error,
             buff_index += 9;
             break;
         case JIT_ERROR_TX_POWER:
+            memcpy((void *)(buff_ack + buff_index), (void *)"\"TX_POWER\"", 10);
```

Comment:
Those `(void *)` aren't needed, but you probably already know that and are just following the original code's conventions.